NotifyUI/NotificationViewController.swift

```diff
@@ -52,26 +52,25 @@ class NotificationViewController: UIViewController, UNNotificationContentExtensi
             // set request specifcs
             let request = try Request(json: requestObject)
             
-            if let signRequest = request.sign {
+            switch request.type {
+            case .ssh(let signRequest):
                 showView(type: .ssh(signRequest.display), deviceName: sessionName)
-            } else if let gitSign = request.gitSign {
-                
-                switch gitSign.git {
+            case .git(let gitSignRequest):
+                switch gitSignRequest.git {
                 case .commit(let commit):
                     showView(type: ContainerType.commit(commit), deviceName: sessionName)
                 case .tag(let tag):
                     showView(type: ContainerType.tag(tag), deviceName: sessionName)
                 }
-                
-            } else {
+            default:
                 throw InvalidNotificationError.requestTypeUnknown(sessionName)
```

Comment:
👍  should not be default, but the NotificationContentUI extension will only be rendered for the autoApproval and approval notification categories and no others.